2012/05/20 17:03:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii".' in D:\xampp\yii\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('gii')
#1 D:\xampp\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\news\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/news/index.php?r=gii
---
2012/05/20 17:36:08 [error] [exception.CException] exception 'CException' with message 'Property "User.salt" is not defined.' in D:\xampp\yii\framework\base\CComponent.php:131
Stack trace:
#0 D:\xampp\yii\framework\db\ar\CActiveRecord.php(144): CComponent->__get('salt')
#1 D:\xampp\htdocs\news\protected\models\User.php(103): CActiveRecord->__get('salt')
#2 D:\xampp\htdocs\news\protected\components\UserIdentity.php(41): User->validatePassword('3368807')
#3 D:\xampp\htdocs\news\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#4 D:\xampp\yii\framework\validators\CInlineValidator.php(43): LoginForm->authenticate('password', Array)
#5 D:\xampp\yii\framework\validators\CValidator.php(197): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#6 D:\xampp\yii\framework\base\CModel.php(158): CValidator->validate(Object(LoginForm), NULL)
#7 D:\xampp\htdocs\news\protected\controllers\SiteController.php(88): CModel->validate()
#8 D:\xampp\yii\framework\web\actions\CInlineAction.php(50): SiteController->actionLogin()
#9 D:\xampp\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#10 D:\xampp\yii\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#11 D:\xampp\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 D:\xampp\yii\framework\web\CWebApplication.php(276): CController->run('login')
#13 D:\xampp\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('site/login')
#14 D:\xampp\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#15 D:\xampp\htdocs\news\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/news/index.php?r=site/login
HTTP_REFERER=http://localhost/news/index.php?r=site/login
---
2012/05/20 17:39:37 [error] [exception.CException] exception 'CException' with message 'Property "User.id" is not defined.' in D:\xampp\yii\framework\base\CComponent.php:131
Stack trace:
#0 D:\xampp\yii\framework\db\ar\CActiveRecord.php(144): CComponent->__get('id')
#1 D:\xampp\htdocs\news\protected\components\UserIdentity.php(45): CActiveRecord->__get('id')
#2 D:\xampp\htdocs\news\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#3 D:\xampp\yii\framework\validators\CInlineValidator.php(43): LoginForm->authenticate('password', Array)
#4 D:\xampp\yii\framework\validators\CValidator.php(197): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 D:\xampp\yii\framework\base\CModel.php(158): CValidator->validate(Object(LoginForm), NULL)
#6 D:\xampp\htdocs\news\protected\controllers\SiteController.php(88): CModel->validate()
#7 D:\xampp\yii\framework\web\actions\CInlineAction.php(50): SiteController->actionLogin()
#8 D:\xampp\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#9 D:\xampp\yii\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#10 D:\xampp\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 D:\xampp\yii\framework\web\CWebApplication.php(276): CController->run('login')
#12 D:\xampp\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('site/login')
#13 D:\xampp\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#14 D:\xampp\htdocs\news\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/news/index.php?r=site/login
HTTP_REFERER=http://localhost/news/index.php?r=site/login
---
2012/05/20 17:39:53 [error] [exception.CException] exception 'CException' with message 'Property "User.id" is not defined.' in D:\xampp\yii\framework\base\CComponent.php:131
Stack trace:
#0 D:\xampp\yii\framework\db\ar\CActiveRecord.php(144): CComponent->__get('id')
#1 D:\xampp\htdocs\news\protected\components\UserIdentity.php(45): CActiveRecord->__get('id')
#2 D:\xampp\htdocs\news\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#3 D:\xampp\yii\framework\validators\CInlineValidator.php(43): LoginForm->authenticate('password', Array)
#4 D:\xampp\yii\framework\validators\CValidator.php(197): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 D:\xampp\yii\framework\base\CModel.php(158): CValidator->validate(Object(LoginForm), NULL)
#6 D:\xampp\htdocs\news\protected\controllers\SiteController.php(88): CModel->validate()
#7 D:\xampp\yii\framework\web\actions\CInlineAction.php(50): SiteController->actionLogin()
#8 D:\xampp\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#9 D:\xampp\yii\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#10 D:\xampp\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 D:\xampp\yii\framework\web\CWebApplication.php(276): CController->run('login')
#12 D:\xampp\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('site/login')
#13 D:\xampp\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#14 D:\xampp\htdocs\news\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/news/index.php?r=site/login
HTTP_REFERER=http://localhost/news/index.php?r=site/login
---
2012/05/21 13:29:13 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in D:\xampp\yii\framework\web\auth\CAccessControlFilter.php:157
Stack trace:
#0 D:\xampp\yii\framework\web\auth\CAccessControlFilter.php(120): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 D:\xampp\yii\framework\web\filters\CFilter.php(39): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 D:\xampp\yii\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#3 D:\xampp\yii\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#4 D:\xampp\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 D:\xampp\yii\framework\web\CController.php(292): CFilterChain->run()
#6 D:\xampp\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 D:\xampp\yii\framework\web\CWebApplication.php(276): CController->run('admin')
#8 D:\xampp\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('user/admin')
#9 D:\xampp\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 D:\xampp\htdocs\news\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/news/index.php/user/admin
HTTP_REFERER=http://localhost/news/index.php/site/login
---
2012/05/21 13:29:50 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in D:\xampp\yii\framework\web\auth\CAccessControlFilter.php:157
Stack trace:
#0 D:\xampp\yii\framework\web\auth\CAccessControlFilter.php(120): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 D:\xampp\yii\framework\web\filters\CFilter.php(39): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 D:\xampp\yii\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#3 D:\xampp\yii\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#4 D:\xampp\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 D:\xampp\yii\framework\web\CController.php(292): CFilterChain->run()
#6 D:\xampp\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 D:\xampp\yii\framework\web\CWebApplication.php(276): CController->run('admin')
#8 D:\xampp\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('user/admin')
#9 D:\xampp\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 D:\xampp\htdocs\news\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/news/index.php/user/admin
HTTP_REFERER=http://localhost/news/index.php/user
---
2012/05/21 13:29:54 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in D:\xampp\yii\framework\web\auth\CAccessControlFilter.php:157
Stack trace:
#0 D:\xampp\yii\framework\web\auth\CAccessControlFilter.php(120): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 D:\xampp\yii\framework\web\filters\CFilter.php(39): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 D:\xampp\yii\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#3 D:\xampp\yii\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#4 D:\xampp\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 D:\xampp\yii\framework\web\CController.php(292): CFilterChain->run()
#6 D:\xampp\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 D:\xampp\yii\framework\web\CWebApplication.php(276): CController->run('admin')
#8 D:\xampp\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('user/admin')
#9 D:\xampp\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 D:\xampp\htdocs\news\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/news/index.php/user/admin
HTTP_REFERER=http://localhost/news/index.php/user
---
2012/05/21 14:29:52 [error] [exception.CException] exception 'CException' with message 'Property "User.user_name" is not defined.' in D:\xampp\yii\framework\base\CComponent.php:131
Stack trace:
#0 D:\xampp\yii\framework\db\ar\CActiveRecord.php(144): CComponent->__get('user_name')
#1 D:\xampp\yii\framework\validators\CStringValidator.php(80): CActiveRecord->__get('user_name')
#2 D:\xampp\yii\framework\validators\CValidator.php(197): CStringValidator->validateAttribute(Object(User), 'user_name')
#3 D:\xampp\yii\framework\base\CModel.php(158): CValidator->validate(Object(User), NULL)
#4 D:\xampp\yii\framework\db\ar\CActiveRecord.php(786): CModel->validate(NULL)
#5 D:\xampp\htdocs\news\protected\controllers\UserController.php(72): CActiveRecord->save()
#6 D:\xampp\yii\framework\web\actions\CInlineAction.php(50): UserController->actionCreate()
#7 D:\xampp\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#8 D:\xampp\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#9 D:\xampp\yii\framework\web\filters\CFilter.php(41): CFilterChain->run()
#10 D:\xampp\yii\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#11 D:\xampp\yii\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#12 D:\xampp\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#13 D:\xampp\yii\framework\web\CController.php(292): CFilterChain->run()
#14 D:\xampp\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 D:\xampp\yii\framework\web\CWebApplication.php(276): CController->run('create')
#16 D:\xampp\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('user/create')
#17 D:\xampp\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#18 D:\xampp\htdocs\news\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/news/index.php/user/create
HTTP_REFERER=http://localhost/news/index.php/user/create
---
2012/05/21 14:41:56 [warning] [application] Failed to set unsafe attribute "status" of "User".
in D:\xampp\htdocs\news\protected\controllers\UserController.php (71)
in D:\xampp\htdocs\news\index.php (13)
2012/05/21 14:41:56 [warning] [application] Failed to set unsafe attribute "join_date" of "User".
in D:\xampp\htdocs\news\protected\controllers\UserController.php (71)
in D:\xampp\htdocs\news\index.php (13)
2012/05/21 14:42:00 [warning] [application] Failed to set unsafe attribute "status" of "User".
in D:\xampp\htdocs\news\protected\controllers\UserController.php (71)
in D:\xampp\htdocs\news\index.php (13)
2012/05/21 14:42:00 [warning] [application] Failed to set unsafe attribute "join_date" of "User".
in D:\xampp\htdocs\news\protected\controllers\UserController.php (71)
in D:\xampp\htdocs\news\index.php (13)
2012/05/21 14:43:00 [warning] [application] Failed to set unsafe attribute "status" of "User".
in D:\xampp\htdocs\news\protected\controllers\UserController.php (71)
in D:\xampp\htdocs\news\index.php (13)
2012/05/21 14:43:00 [warning] [application] Failed to set unsafe attribute "join_date" of "User".
in D:\xampp\htdocs\news\protected\controllers\UserController.php (71)
in D:\xampp\htdocs\news\index.php (13)
2012/05/21 14:45:01 [error] [exception.CException] exception 'CException' with message 'Property "User.password2" is not defined.' in D:\xampp\yii\framework\base\CComponent.php:131
Stack trace:
#0 D:\xampp\yii\framework\db\ar\CActiveRecord.php(144): CComponent->__get('password2')
#1 D:\xampp\yii\framework\web\helpers\CHtml.php(2044): CActiveRecord->__get('password2')
#2 D:\xampp\yii\framework\web\helpers\CHtml.php(1774): CHtml::resolveValue(Object(User), 'password2')
#3 D:\xampp\yii\framework\web\helpers\CHtml.php(1238): CHtml::activeInputField('password', Object(User), 'password2', Array)
#4 D:\xampp\yii\framework\web\widgets\CActiveForm.php(605): CHtml::activePasswordField(Object(User), 'password2', Array)
#5 D:\xampp\htdocs\news\protected\views\user\_form.php(26): CActiveForm->passwordField(Object(User), 'password2', Array)
#6 D:\xampp\yii\framework\web\CBaseController.php(127): require('D:\xampp\htdocs...')
#7 D:\xampp\yii\framework\web\CBaseController.php(96): CBaseController->renderInternal('D:\xampp\htdocs...', Array, true)
#8 D:\xampp\yii\framework\web\CController.php(870): CBaseController->renderFile('D:\xampp\htdocs...', Array, true)
#9 D:\xampp\htdocs\news\protected\views\user\create.php(15): CController->renderPartial('_form', Array)
#10 D:\xampp\yii\framework\web\CBaseController.php(127): require('D:\xampp\htdocs...')
#11 D:\xampp\yii\framework\web\CBaseController.php(96): CBaseController->renderInternal('D:\xampp\htdocs...', Array, true)
#12 D:\xampp\yii\framework\web\CController.php(870): CBaseController->renderFile('D:\xampp\htdocs...', Array, true)
#13 D:\xampp\yii\framework\web\CController.php(783): CController->renderPartial('create', Array, true)
#14 D:\xampp\htdocs\news\protected\controllers\UserController.php(78): CController->render('create', Array)
#15 D:\xampp\yii\framework\web\actions\CInlineAction.php(50): UserController->actionCreate()
#16 D:\xampp\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#17 D:\xampp\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#18 D:\xampp\yii\framework\web\filters\CFilter.php(41): CFilterChain->run()
#19 D:\xampp\yii\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#20 D:\xampp\yii\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#21 D:\xampp\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#22 D:\xampp\yii\framework\web\CController.php(292): CFilterChain->run()
#23 D:\xampp\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#24 D:\xampp\yii\framework\web\CWebApplication.php(276): CController->run('create')
#25 D:\xampp\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('user/create')
#26 D:\xampp\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#27 D:\xampp\htdocs\news\index.php(13): CApplication->run()
#28 {main}
REQUEST_URI=/news/index.php/user/create
HTTP_REFERER=http://localhost/news/index.php/user/create
---
2012/05/21 14:50:17 [warning] [application] Failed to set unsafe attribute "password_repeat" of "User".
in D:\xampp\htdocs\news\protected\controllers\UserController.php (71)
in D:\xampp\htdocs\news\index.php (13)
2012/05/21 14:56:31 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in D:\xampp\yii\framework\web\auth\CAccessControlFilter.php:157
Stack trace:
#0 D:\xampp\yii\framework\web\auth\CAccessControlFilter.php(120): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 D:\xampp\yii\framework\web\filters\CFilter.php(39): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 D:\xampp\yii\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#3 D:\xampp\yii\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#4 D:\xampp\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 D:\xampp\yii\framework\web\CController.php(292): CFilterChain->run()
#6 D:\xampp\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 D:\xampp\yii\framework\web\CWebApplication.php(276): CController->run('admin')
#8 D:\xampp\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('user/admin')
#9 D:\xampp\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 D:\xampp\htdocs\news\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/news/index.php/user/admin
HTTP_REFERER=http://localhost/news/index.php/user/index
---
2012/05/21 14:57:08 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in D:\xampp\yii\framework\web\auth\CAccessControlFilter.php:157
Stack trace:
#0 D:\xampp\yii\framework\web\auth\CAccessControlFilter.php(120): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 D:\xampp\yii\framework\web\filters\CFilter.php(39): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 D:\xampp\yii\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#3 D:\xampp\yii\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#4 D:\xampp\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 D:\xampp\yii\framework\web\CController.php(292): CFilterChain->run()
#6 D:\xampp\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 D:\xampp\yii\framework\web\CWebApplication.php(276): CController->run('admin')
#8 D:\xampp\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('user/admin')
#9 D:\xampp\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 D:\xampp\htdocs\news\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/news/index.php/user/admin
HTTP_REFERER=http://localhost/news/index.php/user/2
---
2012/05/21 14:57:11 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in D:\xampp\yii\framework\web\auth\CAccessControlFilter.php:157
Stack trace:
#0 D:\xampp\yii\framework\web\auth\CAccessControlFilter.php(120): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 D:\xampp\yii\framework\web\filters\CFilter.php(39): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 D:\xampp\yii\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#3 D:\xampp\yii\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#4 D:\xampp\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 D:\xampp\yii\framework\web\CController.php(292): CFilterChain->run()
#6 D:\xampp\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 D:\xampp\yii\framework\web\CWebApplication.php(276): CController->run('delete')
#8 D:\xampp\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('user/delete')
#9 D:\xampp\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 D:\xampp\htdocs\news\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/news/index.php/user/delete/2
HTTP_REFERER=http://localhost/news/index.php/user/2
---
2012/05/21 17:48:49 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in D:\xampp\yii\framework\web\auth\CAccessControlFilter.php:157
Stack trace:
#0 D:\xampp\yii\framework\web\auth\CAccessControlFilter.php(120): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 D:\xampp\yii\framework\web\filters\CFilter.php(39): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 D:\xampp\yii\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#3 D:\xampp\yii\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#4 D:\xampp\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 D:\xampp\yii\framework\web\CController.php(292): CFilterChain->run()
#6 D:\xampp\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 D:\xampp\yii\framework\web\CWebApplication.php(276): CController->run('admin')
#8 D:\xampp\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('user/admin')
#9 D:\xampp\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 D:\xampp\htdocs\news\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/news/index.php/user/admin
HTTP_REFERER=http://localhost/news/index.php/user/index
---
2012/05/27 11:55:45 [error] [exception.CException] exception 'CException' with message 'Property "User.status" is not defined.' in D:\xampp\yii\framework\base\CComponent.php:131
Stack trace:
#0 D:\xampp\yii\framework\db\ar\CActiveRecord.php(144): CComponent->__get('status')
#1 D:\xampp\htdocs\news\protected\views\user\_view.php(20): CActiveRecord->__get('status')
#2 D:\xampp\yii\framework\web\CBaseController.php(127): require('D:\xampp\htdocs...')
#3 D:\xampp\yii\framework\web\CBaseController.php(96): CBaseController->renderInternal('D:\xampp\htdocs...', Array, true)
#4 D:\xampp\yii\framework\web\CController.php(870): CBaseController->renderFile('D:\xampp\htdocs...', Array, true)
#5 D:\xampp\yii\framework\zii\widgets\CListView.php(242): CController->renderPartial('_view', Array)
#6 D:\xampp\yii\framework\zii\widgets\CBaseListView.php(158): CListView->renderItems()
#7 [internal function]: CBaseListView->renderSection(Array)
#8 D:\xampp\yii\framework\zii\widgets\CBaseListView.php(141): preg_replace_callback('/{(\w+)}/', Array, '{summary}?{sort...')
#9 D:\xampp\yii\framework\zii\widgets\CBaseListView.php(126): CBaseListView->renderContent()
#10 D:\xampp\yii\framework\web\CBaseController.php(174): CBaseListView->run()
#11 D:\xampp\htdocs\news\protected\views\user\index.php(17): CBaseController->widget('zii.widgets.CLi...', Array)
#12 D:\xampp\yii\framework\web\CBaseController.php(127): require('D:\xampp\htdocs...')
#13 D:\xampp\yii\framework\web\CBaseController.php(96): CBaseController->renderInternal('D:\xampp\htdocs...', Array, true)
#14 D:\xampp\yii\framework\web\CController.php(870): CBaseController->renderFile('D:\xampp\htdocs...', Array, true)
#15 D:\xampp\yii\framework\web\CController.php(783): CController->renderPartial('index', Array, true)
#16 D:\xampp\htdocs\news\protected\controllers\UserController.php(133): CController->render('index', Array)
#17 D:\xampp\yii\framework\web\actions\CInlineAction.php(50): UserController->actionIndex()
#18 D:\xampp\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#19 D:\xampp\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#20 D:\xampp\yii\framework\web\filters\CFilter.php(41): CFilterChain->run()
#21 D:\xampp\yii\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#22 D:\xampp\yii\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#23 D:\xampp\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#24 D:\xampp\yii\framework\web\CController.php(292): CFilterChain->run()
#25 D:\xampp\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#26 D:\xampp\yii\framework\web\CWebApplication.php(276): CController->run('')
#27 D:\xampp\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('user')
#28 D:\xampp\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#29 D:\xampp\htdocs\news\index.php(13): CApplication->run()
#30 {main}
REQUEST_URI=/news/index.php/user
---
2012/05/27 11:56:59 [error] [exception.CException] exception 'CException' with message 'Property "User.status" is not defined.' in D:\xampp\yii\framework\base\CComponent.php:131
Stack trace:
#0 D:\xampp\yii\framework\db\ar\CActiveRecord.php(144): CComponent->__get('status')
#1 D:\xampp\yii\framework\web\helpers\CHtml.php(1696): CActiveRecord->__get('status')
#2 D:\xampp\yii\framework\zii\widgets\CDetailView.php(210): CHtml::value(Object(User), 'status')
#3 D:\xampp\yii\framework\web\CBaseController.php(174): CDetailView->run()
#4 D:\xampp\htdocs\news\protected\views\user\view.php(28): CBaseController->widget('zii.widgets.CDe...', Array)
#5 D:\xampp\yii\framework\web\CBaseController.php(127): require('D:\xampp\htdocs...')
#6 D:\xampp\yii\framework\web\CBaseController.php(96): CBaseController->renderInternal('D:\xampp\htdocs...', Array, true)
#7 D:\xampp\yii\framework\web\CController.php(870): CBaseController->renderFile('D:\xampp\htdocs...', Array, true)
#8 D:\xampp\yii\framework\web\CController.php(783): CController->renderPartial('view', Array, true)
#9 D:\xampp\htdocs\news\protected\controllers\UserController.php(55): CController->render('view', Array)
#10 [internal function]: UserController->actionView('1')
#11 D:\xampp\yii\framework\web\actions\CAction.php(107): ReflectionMethod->invokeArgs(Object(UserController), Array)
#12 D:\xampp\yii\framework\web\actions\CInlineAction.php(48): CAction->runWithParamsInternal(Object(UserController), Object(ReflectionMethod), Array)
#13 D:\xampp\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#14 D:\xampp\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#15 D:\xampp\yii\framework\web\filters\CFilter.php(41): CFilterChain->run()
#16 D:\xampp\yii\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#17 D:\xampp\yii\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#18 D:\xampp\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#19 D:\xampp\yii\framework\web\CController.php(292): CFilterChain->run()
#20 D:\xampp\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#21 D:\xampp\yii\framework\web\CWebApplication.php(276): CController->run('view')
#22 D:\xampp\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('user/view')
#23 D:\xampp\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#24 D:\xampp\htdocs\news\index.php(13): CApplication->run()
#25 {main}
REQUEST_URI=/news/index.php/user/1
HTTP_REFERER=http://localhost/news/index.php/user
---
2012/05/27 11:58:18 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in D:\xampp\yii\framework\web\auth\CAccessControlFilter.php:157
Stack trace:
#0 D:\xampp\yii\framework\web\auth\CAccessControlFilter.php(120): CAccessControlFilter->accessDenied(Object(WebUser), 'You are not aut...')
#1 D:\xampp\yii\framework\web\filters\CFilter.php(39): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 D:\xampp\yii\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#3 D:\xampp\yii\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#4 D:\xampp\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 D:\xampp\yii\framework\web\CController.php(292): CFilterChain->run()
#6 D:\xampp\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 D:\xampp\yii\framework\web\CWebApplication.php(276): CController->run('admin')
#8 D:\xampp\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('user/admin')
#9 D:\xampp\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 D:\xampp\htdocs\news\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/news/index.php/user/admin
HTTP_REFERER=http://localhost/news/index.php/user/create
---
2012/05/27 12:00:12 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in D:\xampp\yii\framework\web\auth\CAccessControlFilter.php:157
Stack trace:
#0 D:\xampp\yii\framework\web\auth\CAccessControlFilter.php(120): CAccessControlFilter->accessDenied(Object(WebUser), 'You are not aut...')
#1 D:\xampp\yii\framework\web\filters\CFilter.php(39): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 D:\xampp\yii\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#3 D:\xampp\yii\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#4 D:\xampp\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 D:\xampp\yii\framework\web\CController.php(292): CFilterChain->run()
#6 D:\xampp\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 D:\xampp\yii\framework\web\CWebApplication.php(276): CController->run('admin')
#8 D:\xampp\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('user/admin')
#9 D:\xampp\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 D:\xampp\htdocs\news\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/news/index.php/user/admin
HTTP_REFERER=http://localhost/news/index.php/user/create
---
2012/05/27 12:00:39 [error] [exception.CException] exception 'CException' with message 'Property "User.status" is not defined.' in D:\xampp\yii\framework\base\CComponent.php:131
Stack trace:
#0 D:\xampp\yii\framework\db\ar\CActiveRecord.php(144): CComponent->__get('status')
#1 D:\xampp\yii\framework\web\helpers\CHtml.php(2044): CActiveRecord->__get('status')
#2 D:\xampp\yii\framework\web\helpers\CHtml.php(1774): CHtml::resolveValue(Object(User), 'status')
#3 D:\xampp\yii\framework\web\helpers\CHtml.php(1205): CHtml::activeInputField('text', Object(User), 'status', Array)
#4 D:\xampp\yii\framework\web\widgets\CActiveForm.php(575): CHtml::activeTextField(Object(User), 'status', Array)
#5 D:\xampp\htdocs\news\protected\views\user\_search.php(25): CActiveForm->textField(Object(User), 'status')
#6 D:\xampp\yii\framework\web\CBaseController.php(127): require('D:\xampp\htdocs...')
#7 D:\xampp\yii\framework\web\CBaseController.php(96): CBaseController->renderInternal('D:\xampp\htdocs...', Array, true)
#8 D:\xampp\yii\framework\web\CController.php(870): CBaseController->renderFile('D:\xampp\htdocs...', Array, true)
#9 D:\xampp\htdocs\news\protected\views\user\admin.php(37): CController->renderPartial('_search', Array)
#10 D:\xampp\yii\framework\web\CBaseController.php(127): require('D:\xampp\htdocs...')
#11 D:\xampp\yii\framework\web\CBaseController.php(96): CBaseController->renderInternal('D:\xampp\htdocs...', Array, true)
#12 D:\xampp\yii\framework\web\CController.php(870): CBaseController->renderFile('D:\xampp\htdocs...', Array, true)
#13 D:\xampp\yii\framework\web\CController.php(783): CController->renderPartial('admin', Array, true)
#14 D:\xampp\htdocs\news\protected\controllers\UserController.php(148): CController->render('admin', Array)
#15 D:\xampp\yii\framework\web\actions\CInlineAction.php(50): UserController->actionAdmin()
#16 D:\xampp\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#17 D:\xampp\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#18 D:\xampp\yii\framework\web\filters\CFilter.php(41): CFilterChain->run()
#19 D:\xampp\yii\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#20 D:\xampp\yii\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#21 D:\xampp\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#22 D:\xampp\yii\framework\web\CController.php(292): CFilterChain->run()
#23 D:\xampp\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#24 D:\xampp\yii\framework\web\CWebApplication.php(276): CController->run('admin')
#25 D:\xampp\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('user/admin')
#26 D:\xampp\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#27 D:\xampp\htdocs\news\index.php(13): CApplication->run()
#28 {main}
REQUEST_URI=/news/index.php/user/admin
HTTP_REFERER=http://localhost/news/index.php/user/create
---
2012/05/27 12:01:20 [error] [exception.CException] exception 'CException' with message 'Property "User.Id" is not defined.' in D:\xampp\yii\framework\base\CComponent.php:131
Stack trace:
#0 D:\xampp\yii\framework\db\ar\CActiveRecord.php(144): CComponent->__get('Id')
#1 D:\xampp\htdocs\news\protected\models\User.php(96): CActiveRecord->__get('Id')
#2 D:\xampp\htdocs\news\protected\views\user\admin.php(42): User->search()
#3 D:\xampp\yii\framework\web\CBaseController.php(127): require('D:\xampp\htdocs...')
#4 D:\xampp\yii\framework\web\CBaseController.php(96): CBaseController->renderInternal('D:\xampp\htdocs...', Array, true)
#5 D:\xampp\yii\framework\web\CController.php(870): CBaseController->renderFile('D:\xampp\htdocs...', Array, true)
#6 D:\xampp\yii\framework\web\CController.php(783): CController->renderPartial('admin', Array, true)
#7 D:\xampp\htdocs\news\protected\controllers\UserController.php(148): CController->render('admin', Array)
#8 D:\xampp\yii\framework\web\actions\CInlineAction.php(50): UserController->actionAdmin()
#9 D:\xampp\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#10 D:\xampp\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#11 D:\xampp\yii\framework\web\filters\CFilter.php(41): CFilterChain->run()
#12 D:\xampp\yii\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#13 D:\xampp\yii\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#14 D:\xampp\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#15 D:\xampp\yii\framework\web\CController.php(292): CFilterChain->run()
#16 D:\xampp\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#17 D:\xampp\yii\framework\web\CWebApplication.php(276): CController->run('admin')
#18 D:\xampp\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('user/admin')
#19 D:\xampp\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#20 D:\xampp\htdocs\news\index.php(13): CApplication->run()
#21 {main}
REQUEST_URI=/news/index.php/user/admin
HTTP_REFERER=http://localhost/news/index.php/user/create
---
2012/05/27 12:02:08 [error] [exception.CException] exception 'CException' with message 'Property "User.status" is not defined.' in D:\xampp\yii\framework\base\CComponent.php:131
Stack trace:
#0 D:\xampp\yii\framework\db\ar\CActiveRecord.php(144): CComponent->__get('status')
#1 D:\xampp\yii\framework\web\helpers\CHtml.php(2044): CActiveRecord->__get('status')
#2 D:\xampp\yii\framework\web\helpers\CHtml.php(1774): CHtml::resolveValue(Object(User), 'status')
#3 D:\xampp\yii\framework\web\helpers\CHtml.php(1205): CHtml::activeInputField('text', Object(User), 'status', Array)
#4 D:\xampp\yii\framework\zii\widgets\grid\CDataColumn.php(105): CHtml::activeTextField(Object(User), 'status', Array)
#5 D:\xampp\yii\framework\zii\widgets\grid\CGridColumn.php(106): CDataColumn->renderFilterCellContent()
#6 D:\xampp\yii\framework\zii\widgets\grid\CGridView.php(450): CGridColumn->renderFilterCell()
#7 D:\xampp\yii\framework\zii\widgets\grid\CGridView.php(428): CGridView->renderFilter()
#8 D:\xampp\yii\framework\zii\widgets\grid\CGridView.php(398): CGridView->renderTableHeader()
#9 D:\xampp\yii\framework\zii\widgets\CBaseListView.php(158): CGridView->renderItems()
#10 [internal function]: CBaseListView->renderSection(Array)
#11 D:\xampp\yii\framework\zii\widgets\CBaseListView.php(141): preg_replace_callback('/{(\w+)}/', Array, '{summary}?{item...')
#12 D:\xampp\yii\framework\zii\widgets\CBaseListView.php(126): CBaseListView->renderContent()
#13 D:\xampp\yii\framework\web\CBaseController.php(174): CBaseListView->run()
#14 D:\xampp\htdocs\news\protected\views\user\admin.php(55): CBaseController->widget('zii.widgets.gri...', Array)
#15 D:\xampp\yii\framework\web\CBaseController.php(127): require('D:\xampp\htdocs...')
#16 D:\xampp\yii\framework\web\CBaseController.php(96): CBaseController->renderInternal('D:\xampp\htdocs...', Array, true)
#17 D:\xampp\yii\framework\web\CController.php(870): CBaseController->renderFile('D:\xampp\htdocs...', Array, true)
#18 D:\xampp\yii\framework\web\CController.php(783): CController->renderPartial('admin', Array, true)
#19 D:\xampp\htdocs\news\protected\controllers\UserController.php(148): CController->render('admin', Array)
#20 D:\xampp\yii\framework\web\actions\CInlineAction.php(50): UserController->actionAdmin()
#21 D:\xampp\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#22 D:\xampp\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#23 D:\xampp\yii\framework\web\filters\CFilter.php(41): CFilterChain->run()
#24 D:\xampp\yii\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#25 D:\xampp\yii\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#26 D:\xampp\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#27 D:\xampp\yii\framework\web\CController.php(292): CFilterChain->run()
#28 D:\xampp\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#29 D:\xampp\yii\framework\web\CWebApplication.php(276): CController->run('admin')
#30 D:\xampp\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('user/admin')
#31 D:\xampp\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#32 D:\xampp\htdocs\news\index.php(13): CApplication->run()
#33 {main}
REQUEST_URI=/news/index.php/user/admin
HTTP_REFERER=http://localhost/news/index.php/user/create
---
2012/05/27 12:09:13 [error] [exception.CException] exception 'CException' with message 'Property "WebUser.role" is not defined.' in D:\xampp\yii\framework\base\CComponent.php:131
Stack trace:
#0 D:\xampp\yii\framework\web\auth\CWebUser.php(141): CComponent->__get('role')
#1 D:\xampp\htdocs\news\protected\views\layouts\main.php(35): CWebUser->__get('role')
#2 D:\xampp\yii\framework\web\CBaseController.php(127): require('D:\xampp\htdocs...')
#3 D:\xampp\yii\framework\web\CBaseController.php(96): CBaseController->renderInternal('D:\xampp\htdocs...', Array, true)
#4 D:\xampp\yii\framework\web\widgets\CContentDecorator.php(77): CBaseController->renderFile('D:\xampp\htdocs...', Array, true)
#5 D:\xampp\yii\framework\web\widgets\CContentDecorator.php(55): CContentDecorator->decorate('<div id="conten...')
#6 D:\xampp\yii\framework\web\widgets\COutputProcessor.php(45): CContentDecorator->processOutput('<div id="conten...')
#7 D:\xampp\yii\framework\web\CBaseController.php(207): COutputProcessor->run()
#8 D:\xampp\yii\framework\web\CBaseController.php(301): CBaseController->endWidget('CContentDecorat...')
#9 D:\xampp\htdocs\news\protected\views\layouts\column1.php(5): CBaseController->endContent()
#10 D:\xampp\yii\framework\web\CBaseController.php(127): require('D:\xampp\htdocs...')
#11 D:\xampp\yii\framework\web\CBaseController.php(96): CBaseController->renderInternal('D:\xampp\htdocs...', Array, true)
#12 D:\xampp\yii\framework\web\CController.php(785): CBaseController->renderFile('D:\xampp\htdocs...', Array, true)
#13 D:\xampp\htdocs\news\protected\controllers\SiteController.php(32): CController->render('index')
#14 D:\xampp\yii\framework\web\actions\CInlineAction.php(50): SiteController->actionIndex()
#15 D:\xampp\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#16 D:\xampp\yii\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#17 D:\xampp\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 D:\xampp\yii\framework\web\CWebApplication.php(276): CController->run('')
#19 D:\xampp\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('')
#20 D:\xampp\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#21 D:\xampp\htdocs\news\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/news/index.php
HTTP_REFERER=http://localhost/news/index.php/user/admin
---
2012/05/27 12:10:17 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in D:\xampp\yii\framework\web\auth\CAccessControlFilter.php:157
Stack trace:
#0 D:\xampp\yii\framework\web\auth\CAccessControlFilter.php(120): CAccessControlFilter->accessDenied(Object(WebUser), 'You are not aut...')
#1 D:\xampp\yii\framework\web\filters\CFilter.php(39): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 D:\xampp\yii\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#3 D:\xampp\yii\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#4 D:\xampp\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 D:\xampp\yii\framework\web\CController.php(292): CFilterChain->run()
#6 D:\xampp\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 D:\xampp\yii\framework\web\CWebApplication.php(276): CController->run('admin')
#8 D:\xampp\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('user/admin')
#9 D:\xampp\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 D:\xampp\htdocs\news\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/news/index.php/user/admin
HTTP_REFERER=http://localhost/news/index.php/user
---
2012/05/27 12:25:38 [error] [exception.CException] exception 'CException' with message 'Property "WebUser.role" is not defined.' in D:\xampp\yii\framework\base\CComponent.php:131
Stack trace:
#0 D:\xampp\yii\framework\web\auth\CWebUser.php(141): CComponent->__get('role')
#1 D:\xampp\htdocs\news\protected\views\layouts\main.php(35): CWebUser->__get('role')
#2 D:\xampp\yii\framework\web\CBaseController.php(127): require('D:\xampp\htdocs...')
#3 D:\xampp\yii\framework\web\CBaseController.php(96): CBaseController->renderInternal('D:\xampp\htdocs...', Array, true)
#4 D:\xampp\yii\framework\web\widgets\CContentDecorator.php(77): CBaseController->renderFile('D:\xampp\htdocs...', Array, true)
#5 D:\xampp\yii\framework\web\widgets\CContentDecorator.php(55): CContentDecorator->decorate('<div id="conten...')
#6 D:\xampp\yii\framework\web\widgets\COutputProcessor.php(45): CContentDecorator->processOutput('<div id="conten...')
#7 D:\xampp\yii\framework\web\CBaseController.php(207): COutputProcessor->run()
#8 D:\xampp\yii\framework\web\CBaseController.php(301): CBaseController->endWidget('CContentDecorat...')
#9 D:\xampp\htdocs\news\protected\views\layouts\column1.php(5): CBaseController->endContent()
#10 D:\xampp\yii\framework\web\CBaseController.php(127): require('D:\xampp\htdocs...')
#11 D:\xampp\yii\framework\web\CBaseController.php(96): CBaseController->renderInternal('D:\xampp\htdocs...', Array, true)
#12 D:\xampp\yii\framework\web\CController.php(785): CBaseController->renderFile('D:\xampp\htdocs...', Array, true)
#13 D:\xampp\htdocs\news\protected\controllers\SiteController.php(32): CController->render('index')
#14 D:\xampp\yii\framework\web\actions\CInlineAction.php(50): SiteController->actionIndex()
#15 D:\xampp\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#16 D:\xampp\yii\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#17 D:\xampp\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 D:\xampp\yii\framework\web\CWebApplication.php(276): CController->run('')
#19 D:\xampp\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('')
#20 D:\xampp\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#21 D:\xampp\htdocs\news\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/news/index.php
HTTP_REFERER=http://localhost/news/index.php/user
---
2012/05/27 13:25:13 [error] [exception.CException] exception 'CException' with message 'Property "WebUser.roles" is not defined.' in D:\xampp\yii\framework\base\CComponent.php:131
Stack trace:
#0 D:\xampp\yii\framework\web\auth\CWebUser.php(141): CComponent->__get('roles')
#1 D:\xampp\htdocs\news\protected\views\layouts\main.php(35): CWebUser->__get('roles')
#2 D:\xampp\yii\framework\web\CBaseController.php(127): require('D:\xampp\htdocs...')
#3 D:\xampp\yii\framework\web\CBaseController.php(96): CBaseController->renderInternal('D:\xampp\htdocs...', Array, true)
#4 D:\xampp\yii\framework\web\widgets\CContentDecorator.php(77): CBaseController->renderFile('D:\xampp\htdocs...', Array, true)
#5 D:\xampp\yii\framework\web\widgets\CContentDecorator.php(55): CContentDecorator->decorate('<div class="spa...')
#6 D:\xampp\yii\framework\web\widgets\COutputProcessor.php(45): CContentDecorator->processOutput('<div class="spa...')
#7 D:\xampp\yii\framework\web\CBaseController.php(207): COutputProcessor->run()
#8 D:\xampp\yii\framework\web\CBaseController.php(301): CBaseController->endWidget('CContentDecorat...')
#9 D:\xampp\htdocs\news\protected\views\layouts\column2.php(21): CBaseController->endContent()
#10 D:\xampp\yii\framework\web\CBaseController.php(127): require('D:\xampp\htdocs...')
#11 D:\xampp\yii\framework\web\CBaseController.php(96): CBaseController->renderInternal('D:\xampp\htdocs...', Array, true)
#12 D:\xampp\yii\framework\web\CController.php(785): CBaseController->renderFile('D:\xampp\htdocs...', Array, true)
#13 D:\xampp\htdocs\news\protected\controllers\UserController.php(102): CController->render('update', Array)
#14 [internal function]: UserController->actionUpdate('1')
#15 D:\xampp\yii\framework\web\actions\CAction.php(107): ReflectionMethod->invokeArgs(Object(UserController), Array)
#16 D:\xampp\yii\framework\web\actions\CInlineAction.php(48): CAction->runWithParamsInternal(Object(UserController), Object(ReflectionMethod), Array)
#17 D:\xampp\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#18 D:\xampp\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#19 D:\xampp\yii\framework\web\filters\CFilter.php(41): CFilterChain->run()
#20 D:\xampp\yii\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#21 D:\xampp\yii\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#22 D:\xampp\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#23 D:\xampp\yii\framework\web\CController.php(292): CFilterChain->run()
#24 D:\xampp\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#25 D:\xampp\yii\framework\web\CWebApplication.php(276): CController->run('update')
#26 D:\xampp\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('user/update')
#27 D:\xampp\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#28 D:\xampp\htdocs\news\index.php(13): CApplication->run()
#29 {main}
REQUEST_URI=/news/index.php/user/update/1
HTTP_REFERER=http://localhost/news/index.php/user/1
---
2012/05/27 13:54:16 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Invalid request. You cannot update other user's profile.' in D:\xampp\htdocs\news\protected\controllers\UserController.php:106
Stack trace:
#0 [internal function]: UserController->actionUpdate('1')
#1 D:\xampp\yii\framework\web\actions\CAction.php(107): ReflectionMethod->invokeArgs(Object(UserController), Array)
#2 D:\xampp\yii\framework\web\actions\CInlineAction.php(48): CAction->runWithParamsInternal(Object(UserController), Object(ReflectionMethod), Array)
#3 D:\xampp\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 D:\xampp\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#5 D:\xampp\yii\framework\web\filters\CFilter.php(41): CFilterChain->run()
#6 D:\xampp\yii\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#7 D:\xampp\yii\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#8 D:\xampp\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#9 D:\xampp\yii\framework\web\CController.php(292): CFilterChain->run()
#10 D:\xampp\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 D:\xampp\yii\framework\web\CWebApplication.php(276): CController->run('update')
#12 D:\xampp\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('user/update')
#13 D:\xampp\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#14 D:\xampp\htdocs\news\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/news/index.php/user/update/1
HTTP_REFERER=http://localhost/news/index.php/user/1
---
2012/05/27 13:55:47 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Invalid request. You cannot update other user's profile.' in D:\xampp\htdocs\news\protected\controllers\UserController.php:106
Stack trace:
#0 [internal function]: UserController->actionUpdate('1')
#1 D:\xampp\yii\framework\web\actions\CAction.php(107): ReflectionMethod->invokeArgs(Object(UserController), Array)
#2 D:\xampp\yii\framework\web\actions\CInlineAction.php(48): CAction->runWithParamsInternal(Object(UserController), Object(ReflectionMethod), Array)
#3 D:\xampp\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 D:\xampp\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#5 D:\xampp\yii\framework\web\filters\CFilter.php(41): CFilterChain->run()
#6 D:\xampp\yii\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#7 D:\xampp\yii\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#8 D:\xampp\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#9 D:\xampp\yii\framework\web\CController.php(292): CFilterChain->run()
#10 D:\xampp\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 D:\xampp\yii\framework\web\CWebApplication.php(276): CController->run('update')
#12 D:\xampp\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('user/update')
#13 D:\xampp\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#14 D:\xampp\htdocs\news\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/news/index.php/user/update/1
HTTP_REFERER=http://localhost/news/index.php/user/1
---
2012/05/27 16:05:09 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Invalid request. You cannot update other user's profile.' in D:\xampp\htdocs\news\protected\controllers\UserController.php:108
Stack trace:
#0 [internal function]: UserController->actionUpdate('1')
#1 D:\xampp\yii\framework\web\actions\CAction.php(107): ReflectionMethod->invokeArgs(Object(UserController), Array)
#2 D:\xampp\yii\framework\web\actions\CInlineAction.php(48): CAction->runWithParamsInternal(Object(UserController), Object(ReflectionMethod), Array)
#3 D:\xampp\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 D:\xampp\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#5 D:\xampp\yii\framework\web\filters\CFilter.php(41): CFilterChain->run()
#6 D:\xampp\yii\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#7 D:\xampp\yii\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#8 D:\xampp\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#9 D:\xampp\yii\framework\web\CController.php(292): CFilterChain->run()
#10 D:\xampp\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 D:\xampp\yii\framework\web\CWebApplication.php(276): CController->run('update')
#12 D:\xampp\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('user/update')
#13 D:\xampp\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#14 D:\xampp\htdocs\news\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/news/index.php/user/update/1
HTTP_REFERER=http://localhost/news/index.php/user/1
---
2012/05/27 16:35:30 [error] [exception.CException] exception 'CException' with message 'User has an invalid validation rule. The rule must specify attributes to be validated and the validator name.' in D:\xampp\yii\framework\base\CModel.php:287
Stack trace:
#0 D:\xampp\yii\framework\base\CModel.php(259): CModel->createValidators()
#1 D:\xampp\yii\framework\base\CModel.php(302): CModel->getValidators('username')
#2 D:\xampp\yii\framework\web\helpers\CHtml.php(1185): CModel->isAttributeRequired('username')
#3 D:\xampp\yii\framework\web\widgets\CActiveForm.php(560): CHtml::activeLabelEx(Object(User), 'username', Array)
#4 D:\xampp\htdocs\news\protected\views\user\_form-create.php(13): CActiveForm->labelEx(Object(User), 'username')
#5 D:\xampp\yii\framework\web\CBaseController.php(127): require('D:\xampp\htdocs...')
#6 D:\xampp\yii\framework\web\CBaseController.php(96): CBaseController->renderInternal('D:\xampp\htdocs...', Array, true)
#7 D:\xampp\yii\framework\web\CController.php(870): CBaseController->renderFile('D:\xampp\htdocs...', Array, true)
#8 D:\xampp\htdocs\news\protected\views\user\create.php(15): CController->renderPartial('_form-create', Array)
#9 D:\xampp\yii\framework\web\CBaseController.php(127): require('D:\xampp\htdocs...')
#10 D:\xampp\yii\framework\web\CBaseController.php(96): CBaseController->renderInternal('D:\xampp\htdocs...', Array, true)
#11 D:\xampp\yii\framework\web\CController.php(870): CBaseController->renderFile('D:\xampp\htdocs...', Array, true)
#12 D:\xampp\yii\framework\web\CController.php(783): CController->renderPartial('create', Array, true)
#13 D:\xampp\htdocs\news\protected\controllers\UserController.php(79): CController->render('create', Array)
#14 D:\xampp\yii\framework\web\actions\CInlineAction.php(50): UserController->actionCreate()
#15 D:\xampp\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#16 D:\xampp\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#17 D:\xampp\yii\framework\web\filters\CFilter.php(41): CFilterChain->run()
#18 D:\xampp\yii\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#19 D:\xampp\yii\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#20 D:\xampp\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#21 D:\xampp\yii\framework\web\CController.php(292): CFilterChain->run()
#22 D:\xampp\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#23 D:\xampp\yii\framework\web\CWebApplication.php(276): CController->run('create')
#24 D:\xampp\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('user/create')
#25 D:\xampp\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#26 D:\xampp\htdocs\news\index.php(13): CApplication->run()
#27 {main}
REQUEST_URI=/news/index.php/user/create
---
2012/05/27 17:33:38 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Invalid request. Please do not repeat this request again.' in D:\xampp\htdocs\news\protected\controllers\UserController.php:133
Stack trace:
#0 [internal function]: UserController->actionDelete('3')
#1 D:\xampp\yii\framework\web\actions\CAction.php(107): ReflectionMethod->invokeArgs(Object(UserController), Array)
#2 D:\xampp\yii\framework\web\actions\CInlineAction.php(48): CAction->runWithParamsInternal(Object(UserController), Object(ReflectionMethod), Array)
#3 D:\xampp\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 D:\xampp\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#5 D:\xampp\yii\framework\web\filters\CFilter.php(41): CFilterChain->run()
#6 D:\xampp\yii\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#7 D:\xampp\yii\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#8 D:\xampp\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#9 D:\xampp\yii\framework\web\CController.php(292): CFilterChain->run()
#10 D:\xampp\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 D:\xampp\yii\framework\web\CWebApplication.php(276): CController->run('delete')
#12 D:\xampp\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('user/delete')
#13 D:\xampp\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#14 D:\xampp\htdocs\news\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/news/index.php/user/delete/3
HTTP_REFERER=http://localhost/news/index.php/site/login
---
